changelog.d/+removed-copy-target-checks.changed.md

```diff
@@ -0,0 +1,2 @@
+1. mirrord CLI now does not check target type when the `copy_target` feature is enabled. The check is now done only in the operator.
```

Comment:
why skip the client side change?

Reply:
New operators will support rollouts, older operators will not. We can do the check if we extend operator status again (new field for feature flag), but I'm not sure if there's any gain here, since the operator already does this check and returns a nice error

Reply:
Got it, sounds good.